1-org/envs/shared/cai_monitoring.tf

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "kms" {
+  source  = "terraform-google-modules/kms/google"
+  version = "~> 2.1"
+
+  project_id      = module.scc_notifications.project_id
+  keyring         = "krg-cai-monitoring"
+  location        = local.default_region
+  keys            = ["key-cai-monitoring"]
+  prevent_destroy = "false"
```

Comment:
there should be a input similar to https://github.com/terraform-google-modules/terraform-example-foundation/blob/a92e31b3c6e48ca1c13570e96c88042ec65d868a/1-org/envs/shared/variables.tf#L85C11-L85C43
to set this flag

Reply:
this flag also need to be added to the helper script:
- when reading the tfvars file 
- when writing the 1-org tfvars file
- in the validation that checks if the flags to destroy resources has been set

and to the 1-org vars map used  in the integration test to create the test